src/main/java/com/devcycle/sdk/server/local/DVCClient.java

```diff
@@ -100,7 +80,7 @@ public <T> Variable<T> variable(User user, String key, T defaultValue) {
    */
   // TODO: Original return type should match the line below, uncomment once implemented and delete the void return
   // public Map<String, Variable> allVariables(User user) throws DVCException {
-  public void allVariables(User user) throws DVCException {
```

Comment:
why are these getting removed? am I missing that?

Reply:
currently these are only stub methods that don't return anything... once we implement the actual Bucketing Assembly Script code, we'll be fixing these up.. that'll come in a later PR :)